client/src/App.js

```diff
@@ -5,8 +5,8 @@ import "bootstrap/dist/css/bootstrap.css";
 
 import { Route, BrowserRouter as Router, Switch } from "react-router-dom";
 
-import ManageCustomer from "./components/ManageCustomer";
-import ModifyCustomer from "./components/ModifyCustomer";
+import CustomerCreate from "./components/CustomerCreate";
```

Comment:
create is a verb and you want to name this with a noun. For example `CustomerCreateForm`

Reply:
I agree, it also indicates this component should contain a form. I just changed the name to `CustomerCreateForm`.